stories/core/Table.stories.tsx

```diff
@@ -314,14 +314,17 @@ Sortable.args = {
 
 export const Filters: Story<TableProps> = (args) => {
   const { columns, data, ...rest } = args;
-  const onClickHandler = (
-    props: CellProps<{
-      name: string;
-      description: string;
-      ids: number[];
-      date: Date;
-    }>,
-  ) => action(props.row.original.name)();
+  type TableStoryDataType = {
+    index: number;
+    name: string;
+    description: string;
+    ids: number[];
+    date: Date;
+  };
+  const onClickHandler = useCallback(
+    (props: CellProps<TableStoryDataType>) => action(props.row.original.name)(),
+    [],
+  );
```

Comment:
Stuff like this adds up. Maybe we should remove this column

Reply:
Removed.